### PR TITLE
Added jennifer, sam, hermes

### PR DESCRIPTION
**Pull request by imdrasil**
[jennifer](https://github.com/imdrasil/jennifer.cr) 

Added to **ORM/ODM Extensions** section

[hermes](https://github.com/imdrasil/hermes.cr) 

Added to **Search** section

[sam](https://github.com/imdrasil/sam.cr) 

Added to **Task management** section

- [x] Tests pass (`crystal spec`)
- [x] Description of the entry is clear and concise. There is no excessive information like
      **"... for Crystal programming language"**,
      **"... for Crystal"**,
      **"... written in Crystal"** etc.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -310,6 +310,7 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
 ## ORM/ODM Extensions
  * [active_record.cr](https://github.com/waterlink/active_record.cr) - Active Record pattern implementation
  * [crecto](https://github.com/fridgerator/crecto) - Database wrapper, based on Ecto
+ * [jennifer](https://github.com/imdrasil/jennifer.cr) - Active Record pattern implementation with flexible query chainable builder and migration system.
```
Comment (veelenga) [inline README.md]: Please remove full stop for consistency

File: README.md [hunk 2]
```diff
@@ -341,13 +342,15 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
  * [stats](https://github.com/Nephos/stats) - An expressive implementation of statistical distributions
 
 ## Search
+ * [hermes](https://github.com/imdrasil/hermes.cr) - Data Mapper pattern implementation for ElastiSearch.
```
Comment (veelenga) [inline README.md]: And here...

File: README.md [hunk 3]
```diff
@@ -341,13 +342,15 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
...
  * [query-builder](https://github.com/izniburak/query-builder) - Sql Query Builder library
  * [query.cr](https://github.com/waterlink/query.cr) - Query abstraction
  * [soegen](https://github.com/Ragmaanir/soegen) - Elasticsearch client for Crystal similar to the stretcher gem for ruby
 
 ## Task management
  * [crake](https://github.com/MakeNowJust/crake) - Rake-like build utility library. It is just a library, so it does not provide CLI
  * [lake](https://github.com/adlerhsieh/lake) - Rake-like task management for Crystal programs
+ * [sam](https://github.com/imdrasil/sam.cr) - another one Rake-like task manager with namespacing and arguments system.
```
Comment (veelenga) [inline README.md]: Remove full stop and make the sentence uppercase.

**Comment by imdrasil** [comment]
Done

**Comment by veelenga** [comment]
Thx 👍

**Pull request merged**